test/gpu/native/studies/shoc/shoc-sort.chpl

```diff
@@ -363,8 +363,8 @@ proc radixSortBlocks(radixGlobalWorkSize, const nbits : uint(32), const startbit
         // Load keys and vals from Global memory
         var key, value : 4*uint(32);
         const base = 4*i;
-        for param j in 0..3:uint(32) do key[j] = keysIn[base + j];
-        for param j in 0..3:uint(32) do value[j] = valuesIn[base + j];
+        for param j in 0..3 do key[j] = keysIn[base + j];
+        for param j in 0..3 do value[j] = valuesIn[base + j];
```

Comment:
I am not so sure about these changes. Are you sure they are unnecessary? I seem to recall some conversation about needing int(32) for gpu?

Reply:
@e-kayrakli any info here?

Note that `j` is a `param` here and is used purely to index into tuples and arrays, so its type has no impact.

Also, it worked fine prior to #23344, when the type of `j` was `int`, not uint(32).